OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/setupDB.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please fix this copyright line to be accurate and remove "All rights reserved"

Reply:
Issue is fixed, please review.